webapp/templates/purl-create.xhtml

```diff
@@ -56,55 +83,22 @@
                                 </select>
                             </div>
                         </div>
-                        <div class="form-group">
-                            <label>GET content location</label>
-                            <div id="get">
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:copy"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:canonical"></textarea>
-                                <textarea class="pattern form-control" rows="1" property="calli:alternate"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:describedby"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:resides"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:moved"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:missing"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:gone"></textarea>
-                                <textarea class="hidden pattern form-control" rows="1" property="calli:illegal"></textarea>
-                            </div>
-                        </div>
-                        <div class="form-group">
-                            <label for="cache">GET cache control</label>
+                        <div class="form-group col-sm-6">
+                            <label>Target</label>
```

Comment:
What about moving the target field to it's own row (many target URL will be long)? Then move status and cache-control to the same row.